test_runner/src/test/kotlin/ftl/json/MatrixMapTest.kt

```diff
@@ -25,15 +26,16 @@ class MatrixMapTest {
 
     private fun matrixForExecution(executionId: Int): SavedMatrix {
         return SavedMatrix(
-            TestMatrix()
+            matrix = TestMatrix()
                 .setResultStorage(createResultsStorage())
                 .setState(MatrixState.FINISHED)
                 .setTestMatrixId("123")
                 .setTestExecutions(
                     listOf(
                         createStepExecution(executionId, "")
                     )
-                )
+                ),
+            testTimeout = defaultTestTimeout
```

Comment:
maybe make it as a default parameter value

Reply:
If you are thinking about default value for `SavedMatrix` constructor I don't think it's good idea. That would lead us to 2 places where default value (common args and `SavedMatrix` constructor) are hold, easy to forget about one when changing/updating second

Reply:
Now we get testTimeout from api response so we don't need pass them by constructor.